### PR TITLE
Ls 8334 Upgrade Assets listing API to get all assets whose working status changed in a given period

### PR DESCRIPTION
**Pull request by kgaurav1**
Ls 8334 Upgrade Assets listing API to get all assets whose working status changed in a given period

### CODE REVIEW

File: app/com/logistimo/utils/LogistimoUtils.java [hunk 1]
```diff
@@ -233,4 +234,14 @@ public static long convertDateToEpoch(Date date) {
   public static boolean isValidTime(long timestamp) {
     return timestamp > 0 && timestamp <= (System.currentTimeMillis() / 1000) + 43_200;
   }
+
+  public static String getFormattedDate(String date) {
+    SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```
Comment (charanm) [inline app/com/logistimo/utils/LogistimoUtils.java]: Shouldn't we use this format? (UTC) in RFC 3399 format: 2015-05-28T14:07:17Z. We have applied this on all our APIs.

Comment (kgaurav1) [reply app/com/logistimo/utils/LogistimoUtils.java]: We are accepting date from client in RFC 3399 format. We are formatting it in format to query db as date is stored in db in same format.

**Comment by kgaurav1** [comment]
Run analysis

**Comment by kgaurav1** [comment]
Run analysis

**Comment by kgaurav1** [comment]
Run analysis

**Pull request merged**